helm/aws-ebs-volume-tagger/templates/cronjob.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ .Release.Name }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+spec:
+  concurrencyPolicy: Allow
```

Comment:
I'm not sure about having concurrency allowed. There might be some race condition I think. WDYT?

Reply:
good idea I will drop it 👍🏻